devicetypes/zooz/zooz-zse44-temperature-humidity-xs-sensor.src/zooz-zse44-temperature-humidity-xs-sensor.groovy

```diff
@@ -0,0 +1,418 @@
+	tempReportingThreshold: [num:3, title:"Temperature Reporting Threshold", size:1, defaultVal:10, range:"10..100", desc:"10..100 (10 = 1°)"],
+	tempReportingInterval: [num:16, title:"Temperature Reporting Interval", size:2, defaultVal:240, range:"0..480", desc:"0(disabled), 1..480(minutes)"],
+	tempUnit: [num:13, title:"Temperature Unit", size:1, defaultVal:1, options:[0:"Celsius", 1:"Fahrenheit [DEFAULT]"]],
+	tempOffset: [num:14, title:"Temperature Offset", size:1, defaultVal:100, range:"0..200", desc:"0..200 (0: -10°, 100: 0°, 200: +10°)"],
+	highTempThreshold: [num:5, title:"Heat Alert Temperature", size:1, defaultVal:120, range:"50..120", desc:"50..120(°)"],
+	lowTempThreshold: [num:7, title:"Freeze Alert Temperature", size:1, defaultVal:10, range:"10..100", desc:"10..100(°)"],
+	humidityReportingThreshold: [num:4, title:"Humidity Reporting Threshold", size:1, defaultVal:5, range:"1..50", desc:"1..50(%)"],
+	humidityReportingInterval: [num:17, title:"Humidity Reporting Interval", size:2, defaultVal:240, range:"0..480", desc:"0(disabled), 1..480(minutes)"],
+	humidityOffset: [num:15, title:"Humidity Offset", size:1, defaultVal:100, range:"0..200", desc:"0..200 (0: -10%, 100: 0%, 200: +10%)"],
+	highHumidityThreshold: [num:9, title:"High Humidity Alert Level", size:1, defaultVal:0, range:"0..100", desc:"0(disabled), 1..100(%)"],
+	lowHumidityThreshold: [num:11, title:"Low Humidity Alert Level", size:1, defaultVal:0, range:"0..100", desc:"0(disabled), 1..100(%)"]
+]
+
+@Field static Map temperatureSensor = [sensorType:1, scale:1]
+@Field static Map humiditySensor = [sensorType: 5, scale:0]
+@Field static Map temperatureAlarm = [name:"temperatureAlarm", notificationType:4, normalEvent:0, highEvent:2, lowEvent:6]
+@Field static Map humidityAlarm = [name:"humidityAlarm", notificationType:16, normalEvent:0, highEvent:2, lowEvent:6]
+@Field static int wakeUpInterval = 43200
+
+metadata {
+	definition (
+		name: "Zooz ZSE44 Temperature | Humidity XS Sensor",
+		namespace: "Zooz",
+		author: "Kevin LaFramboise (@krlaframboise)",
+		ocfDeviceType:"oic.d.thermostat",
+		vid: "064b47e2-afde-3228-89a4-480f0f55b3fd",
+		mnmn: "SmartThingsCommunity"
+	) {
+		capability "Sensor"
+		capability "Temperature Measurement"
+		capability "Relative Humidity Measurement"
+		capability "Battery"
+		capability "Refresh"
+		capability "Health Check"
+		capability "Configuration"
+		capability "platemusic11009.temperatureHumiditySensor"
+		capability "platemusic11009.temperatureAlarm"
+		capability "platemusic11009.humidityAlarm"
+		capability "platemusic11009.firmware"
+		capability "platemusic11009.syncStatus"
+
+		// zw:Ss2a type:0701 mfr:027A prod:7000 model:E004 ver:1.10 zwv:7.13 lib:03 cc:5E,55,9F,6C sec:86,85,8E,59,31,72,5A,87,73,80,71,70,84,7A
+		fingerprint mfr:"027A", prod:"7000", model:"E004", deviceJoinName: "Zooz Temperature Humidity Sensor" // Zooz ZSE44 Temperature | Humidity XS Sensor
```

Comment:
Please, could You change `deviceJoinName` to "Zooz Multipurpose Sensor" ?

Reply:
I made the change